app/Http/Controllers/V4DB/MangaController.php

```diff
@@ -731,9 +733,27 @@ public function reviews(Request $request, int $id)
             || $this->isExpired($request, $results)
         ) {
             $page = $request->get('page') ?? 1;
-            $manga = $this->jikan->getMangaReviews(new MangaReviewsRequest($id, $page));
-            $response = \json_decode($this->serializer->serialize($manga, 'json'), true);
+            $sort = $request->get('sort') ?? Constants::REVIEWS_SORT_MOST_VOTED;
+
+            if (!in_array($sort, [Constants::REVIEWS_SORT_MOST_VOTED, Constants::REVIEWS_SORT_NEWEST, Constants::REVIEWS_SORT_OLDEST])) {
+                throw new BadRequestException('Invalid sort for reviews. Please refer to the documentation: https://docs.api.jikan.moe/');
+            }
 
+            $spoilers = $request->get('spoilers') ?? false;
+            $preliminary = $request->get('preliminary') ?? false;
+
+            $manga = $this->jikan
+                ->getMangaReviews(
+                    new MangaReviewsRequest(
+                        $id,
+                        $page,
+                        $sort,
+                        $spoilers,
+                        $preliminary
+                    )
+                );
+
+            $response = \json_decode($this->serializer->serialize($manga, 'json'), true);
```

Comment:
This is also repeating, so it should go in it's own function or in a service object. (E.g. "SerializationService")